tests/data/test_meta.py

```diff
@@ -37,6 +37,11 @@ def test_create_get_entry(self, platform: ixmp4.Platform):
             assert entry.value == value
             assert entry.type == type
 
+    def test_illegal_key(self, platform: ixmp4.Platform):
+        run = platform.runs.create("Model", "Scenario")
+        with pytest.raises(InvalidRunMeta):
+            platform.backend.meta.create(run.id, "version", "foo")
+
```

Comment:
```suggestion
  def test_illegal_key(self, platform: ixmp4.Platform):
        run = platform.runs.create("Model", "Scenario")
        for illegal_key in {"model", "scenario", "id", "version", "is_default"}:
            with pytest.raises(InvalidRunMeta):
                platform.backend.meta.create(run.id, illegal_key, "foo")
```

Might be good to test all the keys from a distinct set here so we have to think about what we broke when removing keys from the actual set.

Reply:
Added a loop for testing all illegal keys.